notifications/16102019-tutorialOffer.yml

```diff
@@ -0,0 +1,9 @@
+requirements:
+    local_config:
+        - startup/launch-count<5
+icon: "canvas/icons/announcement.png"
```

Comment:
This icon is currently still a placeholder actually... Change it to `canvas/icons/statistics-request.png`, which is the Orange icon.